conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json

```diff
@@ -194,6 +200,12 @@
       ],
       "datasource": null,
```

Comment:
I think there are a few examples of datasource: null instead of datasource: nuodb